pkg/disk/cloud.go

```diff
@@ -521,6 +521,24 @@ func getDisk(diskID string, ecsClient *ecs.Client) []ecs.Disk {
 	return diskResponse.Disks.Disk
 }
 
+func tagDiskUserTags(diskID string, ecsClient *ecs.Client, tags map[string]string) {
+	addTagsReq := ecs.CreateAddTagsRequest()
+	var userTags []ecs.AddTagsTag
```

Comment:
use `make([]ecs.AddTagsTag, 0, len(tags))` should be more efficient.

Reply:
okay